src/sentry/snuba/discover.py

```diff
@@ -302,33 +381,6 @@ def transform_deprecated_functions_in_columns(columns):
     return new_list, translations
 
 
-def transform_deprecated_functions_in_orderby(orderby):
```

Comment:
Do we not need to handle old functions in the order by anymore?

Reply:
The API sends the orderby already aliased and we enforce that the orderby must be in the selected columns, so I don't actually need a full function to do this. I can use the alias mapping generated from the select columns to change the orderby alias into the new alias if necessary.

Reply:
ok.